mailgun/route_handlers.py

```diff
@@ -161,7 +173,42 @@ def handle_mailing_list_email_route(request):
             )
             ml.send_mail(
                 sender_address.display_name, sender_address.address,
-                member_address, subject, text=body_plain, html=body_html
+                member_address, subject, text=body_plain, html=body_html,
+                attachments=attachments, inlines=inlines
             )
 
     return JsonResponse({'success': True})
+
+
+def _get_attachments_inlines(request):
+    attachments = []
+    inlines = []
+
+    try:
+        attachment_count = int(request.POST.get('attachment-count', 0))
+    except RuntimeError:
+        logger.exception('Unable to determine if there were attachments to '
+                         'this email')
+        attachment_count = 0
+
+    try:
+        content_id_map = json.loads(request.POST.get('content-id-map', '{}'))
+    except RuntimeError:
+        logger.exception('Unable to find content-id map in this email, '
+                         'forwarding all files as attachments.')
+        content_id_map = {}
+    attachment_name_to_cid = {v: k.strip('<>')
+                                  for k,v in content_id_map.iteritems()}
+    logger.debug('Attachment name to cid: {}'.format(attachment_name_to_cid))
+
+    for n in xrange(1, attachment_count+1):
+        attachment_name = 'attachment-{}'.format(n)
+        file_ = request.FILES[attachment_name]
+        if attachment_name in attachment_name_to_cid:
+            file_.cid = attachment_name_to_cid[attachment_name]
+            file_.name = file_.name.replace(' ', '_')
```

Comment:
There's no documentation of what is acceptable to differing email providers...one of outlook/gmail cared about the spaces, the other one didn't.  There's an [RFC](https://tools.ietf.org/html/rfc2392), but the format it describes doesn't match what i've seen generated in the wild by gmail/yahoo/outlook.